intervals/main_objs.py

```diff
@@ -1417,66 +1565,48 @@ def homorhythm(self):
         gets syllables at every offset, and identifies passages where more than two voices are singing the same lyrics_hr
         checks the number of active voices (thus eliminating places where some voices have rests)
         """
+        # active version with lyric ngs
         nr = self.notes()
         dur = self.durations(df=nr)
         ng = self.ngrams(df=dur, n=2)
         dur_ngrams = []
-
-        # find passages with more than 2 active voices
         for index, rows in ng.iterrows():
```

Comment:
This seems like it could be a little bottleneck. It's fine to leave as-is but if speed becomes an issue this is one thing we could revise.

Reply:
I will watch that.  Have not noticed problems